packages/contracts/contracts/voting/token/TokenVoting.sol

```diff
@@ -26,26 +26,14 @@ contract TokenVoting is MajorityVotingBase {
     /// @notice Initializes the component.
     /// @dev This method is required to support [ERC-1822](https://eips.ethereum.org/EIPS/eip-1822).
     /// @param _dao The IDAO interface of the associated DAO.
-    /// @param _supportThreshold The support threshold in percent.
-    /// @param _minParticipation The minimum participation ratio in percent.
-    /// @param _minDuration The minimal duration of a vote in seconds.
-    /// @param _minProposerVotingPower The minimal voting power needed to create a proposal.
+    /// @param _majorityVotingSettings The majority voting settings.
     /// @param _token The [ERC-20](https://eips.ethereum.org/EIPS/eip-20) token used for voting.
     function initialize(
         IDAO _dao,
-        uint64 _supportThreshold,
-        uint64 _minParticipation,
-        uint64 _minDuration,
-        uint256 _minProposerVotingPower,
+        MajorityVotingSettings calldata _majorityVotingSettings,
         IVotesUpgradeable _token
     ) public initializer {
-        __MajorityVotingBase_init(
-            _dao,
-            _supportThreshold,
-            _minParticipation,
-            _minDuration,
-            _minProposerVotingPower
-        );
+        __MajorityVotingBase_init(_dao, _majorityVotingSettings);
```

Comment:
Same topic, plugin settings vs voting setting + others

Reply:
Fixed in c69aa38770f33a15baed838deca116c0c78f7e22